dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java

```diff
@@ -202,14 +204,29 @@ protected void versionTagHelp() {
         out("Description:");
         out("  Update certain fields for a given " + getEntryType().toLowerCase() + " version.");
         out("");
+        printRequiredParametersForVersionTag();
+        printHelpFooter();
+    }
+
+    private void printRequiredParametersForVersionTag() {
         out("Required Parameters:");
         out("  " + ENTRY + " <entry>                                      Complete " + getEntryType().toLowerCase()
-            + " path in Dockstore (ex. quay.io/collaboratory/seqware-bwa-workflow)");
+                + " path in Dockstore (ex. quay.io/collaboratory/seqware-bwa-workflow)");
```

Comment:
If this is a workflow, example should probably be a workflow ID example 
(like `github.com/collaboratory/seqware-bwa-workflow` above)

Reply:
I have changed it to `github.com/DataBiosphere/topmed-workflows/UM_variant_caller_wdl`, which is [this workflow](https://qa.dockstore.org/workflows/github.com/DataBiosphere/topmed-workflows/UM_variant_caller_wdl:1.32.0?tab=launch). Thanks for catching this.